packages/workflows/src/api/WorkflowValidator.ts

```diff
@@ -68,42 +68,6 @@ export class WorkflowValidator {
         }
     }
 
-    /**
-     * Validate supplied parameter
-     * @static
-     * @param {string} path - string to check if contains USS path or DSNAME
-     * @param {string} errorMsg - message to show in case validation fails
-     * @memberof WorkflowValidator
-     * TODO maybe also validation if the file or DSNAME exists on Mainframe
-     */
-    public static validatePath(path: string, errorMsg: string) {
```

Comment:
I get it that people often do not use the CLI APIs that we provide, but we should be more careful when removing functionality from the project.

Reply:
Validation APIs were never exported